content/en/service_management/app_builder/build.md

```diff
@@ -76,13 +76,31 @@ UI components can trigger reactions on an [Event][11].
 
 You can use [JavaScript Expressions][13] anywhere in App Builder to create custom interactions between the different parts of your app.
 
-### Interact with an app in JSON
 
-#### Edit an app
+## View app version history
+
+App Builder keeps a record of every saved version of your app.
+
+To view the version history for your app, in the left-hand menu of your app, click the version history icon {{< img src="service_management/app_builder/version-history-icon.png" inline="true">}}.
+
+The UI displays up to 50 saved or published versions of your app, along with the icon of the user who saved or published the version:
+
+{{< img src="service_management/app_builder/version-history-example.png" alt="An example App Builder version history list with two items, the current version and a previous version" style="width:70%;" >}}
+
+You can perform the following operations:
+
+* To view an app version, click the version in the list.
+* To restore an app version, select the version, then click **Restore Version** in the upper-right.
+* To clone an app version, select the version, then click **Clone Version** in the upper-right.
```

Comment:
These bullets are of the form "To do X, click the button that says do X", but do not describe what X is.
IMO, it's currently not adding any information and could be reduced to simply stating "From here you can view, restore, and clone versions"

If we want to explain the operations more we can make clear that `restoring` changes **this app** in place, while `cloning` makes a **new** app and copies over to it.